zest/releaser/utils.py

```diff
@@ -32,6 +32,7 @@
 MUST_CLOSE_FDS = not sys.platform.startswith('win')
 
 AUTO_RESPONSE = False
+BASE = "."
```

Comment:
This should be `os.curdir`. Well, it seems to be the same everywhere (https://docs.python.org/2/library/os.html#os.curdir) but seems better to me anyway.